src/core/src/bootstrap/Constants.py

```diff
@@ -76,6 +77,9 @@ class ConfigSettings(EnumBackport):
         ASSESSMENT_MODE = 'assessmentMode'
         MAXIMUM_ASSESSMENT_INTERVAL = 'maximumAssessmentInterval'
 
+    # todo: best place to enlist this?
+    ARTIFACT_IDENTIFIER_FOR_ITEMS_TO_CLEANUP_UNDER_TEMP_FOLDER = "*.list"
```

Comment:
This is best set as a list (but with just one entry at this point of time). The code can iterate over a list.

Side note: remove to do. Suggested name for constant to reduce verbosity (and to help with skimming code quickly by putting the most-important pieces first): TEMP_FOLDER_CLEANUP_ARTIFACT_LIST

Reply:
Resolved